add_effect_ppt

6gsn opened the pull request:
robotUIとgraphUIの演出を追加&発表資料を追加

6gsn committed:
update_UI

graphUIを追加/robotUIに動く円を追加/ofxFboBlurを使ってBlur効果を試す

6gsn committed:
add_ppt

発表資料を追加

6gsn committed:
edit_mask

mask.jpgを修正

6gsn committed:
add_animation

Animatableを追加

SakuragiYoshimasa commented:
https://vimeo.com/96569566
この辺とかvimeoでいい感じのアニメーション探して実装してみるいいかも

SakuragiYoshimasa commented:
もう少しいい感じになってきたらマージする！
１つのシーンしかないからあれやけどもう少しメリハリみたいなのつけたほうがいいかも
音のエフェクトとかアニメーションの切り替えとか
意味のあるUIの表示とか
確かに発想はいいしfaceTrackingできてるのはいいけど現時点ではそれだけって印象
少しaddon頼みになってるからもう少し自分で考えたアイデアを自力で実装するといいかも
コードはかけるからaddonなかったら無理みたいになってほしくない！

SakuragiYoshimasa commented:
@ByeongseonPark このままやと一つのプロダクトとして成立するか危ういからプレゼンでよく見せる必要ありやけど出来れば見るだけですげーって思わせて欲しい

ByeongseonPark commented:
バイト帰りで今確認できました！
コメントありがとう！インパクトにかける部分をなんとか解決したい・・・
今実装できた部分をうまく生かして、映画のようにストーリを入れてみようと思う！
残りの時間を踏まえて爆死しないように注意しながら頑張ります

SakuragiYoshimasa commented:
yes!たぶんパックンは出来ることいっぱいあるやろし頑張ってほしい！

6gsn committed:
add_SceneMode&add_particleScene

MovieとFaceTrackerで実装したものを切り替えながら、PVっぽく作る仕様に変更
Sceneコントルールを追加
まずparticleを目線で動かすシーンを追加準備

ByeongseonPark commented:
大きすぎるファイルをコミットしたまま作業を進めていたせいで、Syncが効かなくなってしまったので一度マージさせて修正します！
